lib/readline.js

```diff
@@ -723,12 +723,12 @@ Interface.prototype._historyPrev = function() {
 
 // Returns the last character's display position of the given string
 Interface.prototype._getDisplayPos = function(str) {
-  var offset = 0;
+  let offset = 0;
   const col = this.columns;
-  var row = 0;
-  var code;
+  let row = 0;
+  let code;
```

Comment:
nit: the variable `code` can be moved inside for loop on line 732
```javascript
const code = str.codePointAt(i);
```

Reply:
Hi! I did this too (commit amended). Thanks.